integr8ly/integreatly-operator
# MGDAPI-4654 : Move and rename file

# Issue link

https://issues.redhat.com/browse/MGDAPI-4654

# What

Two PR's merged around the same time one adding this file and one changing the layout of the folder. This fixes the issue.

# Verification steps



* clone locally
* run `make mkdocs/serve`
* Expected: no error messages in logs
* Open the web page on http://127.0.0.1:8000/
* In side page under the testing section there should be a page `Testing Alerts`
* Navigate to `Testing Alerts`
* Expected: page loads with on errors.
